CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to observiq-otel-collector
+
+## Pull Requests
+
+### How to Submit Pull Requests
+
+Everyone is welcome to contribute code to `observiq-otel-collector` via GitHub pull requests (PRs).
+
+To create a new PR, fork the project in GitHub and clone the upstream repo:
+
+```sh
+$ git clone https://github.com/observiq/observiq-otel-collector
+```
+
+This would put the project in the `observiq-otel-collector` directory in current working directory.
+
+Enter the newly created directory and add your fork as a new remote:
+
+```sh
+$ git remote add <YOUR_FORK> git@github.com:<YOUR_GITHUB_USERNAME>/observiq-otel-collector
+```
+
+Check out a new branch, make modifications, run linters and tests, and push the branch to your fork:
+
+```sh
+$ git checkout -b <YOUR_BRANCH_NAME>
+# edit files
+$ make test
+$ git add
+$ git commit
+$ git push <YOUR_FORK> <YOUR_BRANCH_NAME>
+```
+
+Open a pull request against the main `observiq-otel-collector` repo.
+
+#### Commands to run before submitting PR
+
+Our CI runs the following checks on each PR. You can run the following local commands to ensure your code is ready for PR:
+
+- Build (`make collector`)
+- CI Checks (`make ci-checks`)
+
+
+### How to Receive Comments
```

Comment:
Feels like this section should be "How to Receive Feedback" or perhaps the first "if the PR is not ready for review" just doesn't quite feel like it fits.